packages/pyroscope-flamegraph/src/format/format.ts

```diff
@@ -101,10 +101,12 @@ class DurationFormatter {
       nStr = '< 0.01';
     }
 
-    return `${nStr} ${
-      this.units ||
-      `${this.suffix}${n === 1 || this.suffix.length === 2 ? '' : 's'}`
-    }`;
+    return withUnits
+      ? `${nStr} ${
+          this.units ||
+          `${this.suffix}${n === 1 || this.suffix.length === 2 ? '' : 's'}`
+        }`
+      : nStr;
   }
```

Comment:
What's going on here? Can you add a test to demonstrate this logic? https://github.com/pyroscope-io/pyroscope/blob/59d34aaf79d7a8a873d29543cbc722d8fffd4896/packages/pyroscope-flamegraph/src/format/format.spec.ts

Reply:
sure, but that's the part of different PR